src/node.js

```diff
@@ -340,7 +340,8 @@
     function _tickCallback() {
       var callback, threw, tock;
 
-      scheduleMicrotasks();
+      if (microtasksScheduled)
```

Comment:
Oh, I thought it was `if (!microtasksScheduled)`. So it's basically the same as if you remove `scheduleMicrotasks`